pkg/organization/organization.go

```
@@ -35,6 +35,28 @@ func GetOrganizations(key string, limit int, fetchLabels bool, fetchCapabilities
 	return organizations.Items().Slice(), nil
 }
 
+func GetOrganization(orgID string, conn *sdk.Connection) (*v1.Organization, error) {
+	orgResponse, err := conn.AccountsMgmt().V1().Organizations().Organization(orgID).Get().Send()
+	if err != nil {
+		return nil, fmt.Errorf("can't retrieve organization: %w", err)
+	}
+
+	return orgResponse.Body(), nil
+}
+
+func AddLabel(orgID string, key string, value string, isInternal bool, conn *sdk.Connection) (*v1.Label, error) {
+	var lbl *v1.Label
+	var err error
+	if lbl, err = label.CreateLabel(key, value, isInternal); err != nil {
+		return nil, fmt.Errorf("%v", err)
+	}
+	lblResponse, err := conn.AccountsMgmt().V1().Organizations().Organization(orgID).Labels().Add().Body(lbl).Send()
+	if err != nil {
+		return nil, fmt.Errorf("can't add new label: %w", err)
+	}
+	return lblResponse.Body(), err
```

let's return the error as nil